core/src/main/java/tech/ydb/core/impl/call/ReadWriteStreamCall.java

```diff
@@ -28,10 +28,11 @@
  * @param <W> type of message to be sent to the server
  */
 public class ReadWriteStreamCall<R, W> extends ClientCall.Listener<R> implements GrpcReadWriteStream<R, W> {
-    private static final Logger logger = LoggerFactory.getLogger(GrpcTransport.class);
+    private static final Logger logger = LoggerFactory.getLogger(ReadWriteStreamCall.class);
```

Comment:
Looks like a copy-paste error in a logger definition

Reply:
Could sometimes be intended to setup logging level and/or filter logs by a single name (like "GrpcTransport") to see whats happening on this level. Not sure about this particular case

Reply:
Yes, it is not a copy-paste error, is was made for filtering logs by one class name

Reply:
Restored and commented previous value